src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
```

Comment:
[Checklist](https://github.com/mate-academy/react_person/blob/master/checklist.md) p.1 use destructuring. Check all over the document.